src/pages/home/report/ReportActionCompose.js

```diff
@@ -177,7 +178,6 @@ class ReportActionCompose extends React.Component {
         this.submitForm = this.submitForm.bind(this);
         this.setIsFocused = this.setIsFocused.bind(this);
         this.setIsFullComposerAvailable = this.setIsFullComposerAvailable.bind(this);
-        this.focus = this.focus.bind(this);
```

Comment:
Please make sure that after this removal, it won't cause crash.
There might be case when `setUpComposeFocusManager` / `onComposerFocus` callback is called before `setTextInputRef` is called.

Reply:
So in this case it will be safer to revert this change. Because the `focusWithDelay.js` would handle this scenario, when the `textInput` is not set:

https://github.com/Expensify/App/blob/6586dbf59e7a61c1da489ef2cb8f08d1fbbca032/src/libs/focusWithDelay/focusWithDelay.js#L22-L25

Reply:
Sure. As I said, this component is being refactored so I don't have too much concern about this

Reply:
Reverted :) 

Yeah, but I thought that the PR  was waiting for this change, but now we know that this might result in some issues.